spec/requests/doorkeeper/authorization_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+RSpec.describe "GET /oauth/authorize" do
+  include Warden::Test::Helpers
+
+  let!(:doorkeeper_app) { create :doorkeeper_application, redirect_uri: "https://localhost:34343/auth/callback" }
+  let!(:resource_owner) { create :user }
+
+  before do
+    Warden.test_mode!
+    login_as(resource_owner, scope: :user)
+  end
+
+  def authorize_url
+    oauth_authorization_url(
+      client_id: doorkeeper_app.uid,
+      redirect_uri: doorkeeper_app.redirect_uri,
+      response_type: "code",
+      state: "c987b113ab6ea39c642f712ba587e8975c05d4fa7bd36fd7"
```

Comment:
Is it possible to change this state to be something that's obviously less meaningful? I always worry about these sorts of long variables in the code. Are they 'magic'? Are they a reference to some other piece of the code? Or are we giving away secret values unintentionally?

If it's irrelevant, then perhaps set it to state: 'irrelevant'?